### DISCUSSION TITLE
Where is able to download modules jar?

### DISCUSSION THREAD

**Discussion by BadWolf-i** [TheNewEconomy/TNE-Bukkit#512] [Q&A] [answered]
I looking for download link for THE modules jar. Can you please help me?

**Comment by BadWolf-i** [comment, marked as answer]
Also where i can read abound module? How to configurate or so? 
![image](https://user-images.githubusercontent.com/31863578/188720372-37432bb6-1d0c-46f5-ae50-67b3e44a1309.png)

**Reply by creatorfromhell** [reply]
The ones you've highlighted aren't released as you'll notice under TNE version it says TBA. Other than that the available ones are on the discord for download.

**Reply by BadWolf-i** [reply]
Thanks a lot!
